modules/create-vm/pkg/vmcreator/vm-creator.go

```diff
@@ -80,10 +85,46 @@ func (v *VMCreator) CreateVM() (*kubevirtv1.VirtualMachine, error) {
 		return v.createVMFromTemplate()
 	case constants.VMManifestCreationMode:
 		return v.createVMFromManifest()
+	case constants.VirtctlCreatingMode:
+		return v.createVMVirtctl()
 	}
 	return nil, zerrors.NewMissingRequiredError("unknown creation mode: %v", v.cliOptions.GetCreationMode())
 }
 
+func (v *VMCreator) createVMVirtctl() (*kubevirtv1.VirtualMachine, error) {
+	var vm kubevirtv1.VirtualMachine
+	output := &bytes.Buffer{}
+
+	cmd := createCommand(v.cliOptions.Virtctl, output)
```

Comment:
Create command like it is done here? https://github.com/kubevirt/kubevirt/blob/3c25c8606e8aac1fcb31a73a3f444ebfffb27d20/tests/clientcmd/virtctl.go#L53